doc/developer-guides/images/trusty-boot-flow.dot

```diff
@@ -2,9 +2,10 @@ digraph G {
    rankdir=LR;
    rank=same;
    bgcolor="transparent";
-   uosl1 [label="UOS_Loader"]
+   uservml1 [label="User VM OS Bootloader"]
```

Comment:
Isn't it nice to be able to edit these images like this :)
I think this image would look better if we put a newline in these labels so the oval isn't so long
```suggestion
   uservml1 [label="User VM OS\nBootloader"]
```